src/HotChocolate/Core/test/Types.Scalars.Tests/__snapshots__/LongitudeTypeTests.Longitude_Integration.snap

```diff
@@ -0,0 +1,5 @@
+{
+  "data": {
+    "test": 0
```

Comment:
@michaelstaib  This is a difficult one. The input is `"180° 0' 0.000\" E"` the output is 180.0. Does this make sense?

Reply:
We’ll serialize it back